sourced/ml/models/tensorflow_model.py

```diff
@@ -0,0 +1,69 @@
+from modelforge import register_model, Model
+
+
+@register_model
+class TensorFlowModel(Model):
+    """
+    TensorFlow Protobuf model exported in the ASDS format with GraphDef inside.
+    """
+    NAME = "tensorflow-model"
+
+    def construct(self, tensorflow_model=None, session=None, list_outputs=None):
```

Comment:
Remove `tensorflow_model`. You can take it from the session.
`session: "tensorflow.Session"`
`list_outputs` > `outputs: List[str]`

Mind the strings - we do not want to rigidly depend on Tensorflow.